roles/provision-keycloak-apb/templates/configmap.yml.j2

```diff
@@ -1,17 +1,16 @@
 apiVersion: v1
-kind: Secret
+kind: ConfigMap
 metadata:
   name: keycloak
   namespace: {{ namespace }}
   labels:
     name: keycloak
     mobile: enabled
-stringData:
+    serviceName: keycloak
+data:
   type: keycloak
   realm: {{namespace}}
   name: keycloak
```

Comment:
@philbrookes this seems redundant. Does the mobile-cli need the name as a label and in metadata?

Reply:
@pb82 this isn't metadata, this is the content of the secret/configmap. Though whether it is still required I'm unsure, this used to be required for the mcp-api-server to figure out what to call the service, but that may no longer be required in the mobile-cli. @maleck13 any thoughts?

Reply:
@pb82 oh my mistake, I see that it is listed in both metadat and labels now, the location of this comment threw me off!

Yes, I think just having one of these would be sufficient.

Reply:
Yes I think we can largely work from the service instances now so likely don't need the additional name